eng/pipelines/libraries/build-test-job.yml

```diff
@@ -1,86 +0,0 @@
-parameters:
```

Comment:
This will indeed break runtimelab as we are using this template. This reminds me that I should really finish up the work on runtimelab.yml so that it gets added to dotnet/runtime as well and changes like this one should fix up not only runtime.yml but also the necessary changes for runtimelab.yml as well.

Reply:
Right, we should just build the libraries tests as part of the product build in `runtimelab.yml`. 

The reason for that is because @ViktorHofer will be moving tests to use P2P to source projects and that would cause the test build if in a separate machine to build all the sources again or to upload all the artifacts directory and download them on the test build which is not ideal.